conf/icr_alma.config

```diff
@@ -0,0 +1,54 @@
+
+/*
+    * -------------------------------------------------
+    *  Nextflow nf-core config file for ICR alma HPC
+    * -------------------------------------------------
+    * Defines slurm process executor and singularity
+    * settings.
+    *
+    */
+params {
+
+    config_profile_description = "Nextflow nf-core profile for ICR alma HPC"
+    config_profile_contact = "Rachel Alcraft (@rachelicr), Mira Sarkis (@msarkis-icr)"
+}
+
+process {
+    queue="compute"
+    executor = "slurm"
+    maxRetries    = 3
+    maxErrors     = '-1'
+    clusterOptions = '--mem-per-cpu=8000'
+
+    max_memory = '256.GB'
```

Comment:
Nax_menory seems to be duplicated now, and I just released the max_ parameters are in the wrong place - they should be in `params`

Resourcelimits are fine where they are though @rachelicr